src/Symfony/Component/EventDispatcher/EventDispatcher.php

```diff
@@ -229,14 +229,6 @@ public function removeSubscriber(EventSubscriberInterface $subscriber)
      * @param object     $event     The event object to pass to the event handlers/listeners
      */
     protected function callListeners(iterable $listeners, string $eventName, $event)
-    {
-        $this->doDispatch($listeners, $eventName, $event);
-    }
-
-    /**
-     * @deprecated since Symfony 4.3, use callListeners() instead
-     */
-    protected function doDispatch($listeners, $eventName, Event $event)
     {
         $stoppable = $event instanceof Event || $event instanceof ContractsEvent || $event instanceof StoppableEventInterface;
```

Comment:
only the first check can be true right now due to the `Event` typehint on `$event`